.github/CHANGELOG.md

```diff
@@ -125,6 +125,33 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 * PennyLane's test suite is now code-formatted using `black -l 100`.
   [(#1222)](https://github.com/PennyLaneAI/pennylane/pull/1222)
 
+* `qml.Hamiltonian` objects can now be recorded inside quantum tapes. 
+  [(#1142)](https://github.com/PennyLaneAI/pennylane/pull/1142)
```

Comment:
@Lucaman99 I would _probably_ remove this one -- it is more an implementation detail of the entry below